File: Ad-Hoc/URI 1893.rb
line = gets.split(' ')
inicio = line[0].to_i
final = line[1].to_i

if inicio >= 0 && final <= 2
	puts "nova"
elsif final > inicio && final <= 96
	puts "crescente"
elsif inicio >= final && final <= 96
	puts "minguante"
else 
	puts "cheia"
end

File: Ad-Hoc/URI 1893.php
<?php
    $line = explode(" ", trim(fgets(STDIN)));
    $inicio = intval($line[0]);
    $final = intval($line[1]);
    
    if ($inicio >= 0 && $final <= 2) {
        echo "nova\n";
    } else if ($final > $inicio && $final <= 96) {
        echo "crescente\n";
    } else if ($inicio >= $final && $final <= 96) {
        echo "minguante\n";
    } else { 
        echo "cheia\n";
    }
?>